components/AppBar.tsx

```diff
@@ -39,22 +36,22 @@ const AppBar = (props: AppBarProps) => {
     relevantAlerts =  alerts.filter(alert => alertInLocation(alert, {latitude:lat, longitude:lon}))
   }
 
-  return (
+  return useMemo(() => (
```

Comment:
Not sure why you use useMemo() here? If it is not really necessary, I think it might be best to skip, since it makes the code a bit harder to read.

Reply:
Will drop it.